### PR TITLE
Merge X-linked spermatogenic failure 1 with Sertoli cell-only syndrome

### LINKED ISSUE #7409: [Merge X-linked spermatogenic failure 1 with Sertoli cell-only syndrome
X-linked spermatogenic failure 1 MONDO:0056795

You have both terms but they are synonyms.

Sertoli cell-only syndrome

Thank you, Gioconda

### PR DESCRIPTION
**Pull request by katiermullen**
Closes #7409

### CODE REVIEW

**Comment by katiermullen** [comment]
@sabrinatoro do you have time to review this PR that we discussed with Ada? If not, I will assign it to another reviewer. Thank you!

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -531789,21 +531769,35 @@ is_a: MONDO:0016120 ! myotonic syndrome
 [Term]
 id: MONDO:0056795
 name: X-linked spermatogenic failure 1
+subset: gard_rare {source="MONDO:GARD"}
 subset: gard_rare {source="GARD:8406", source="MONDO:GARD"}
+subset: nord_rare {source="MONDO:NORD"}
+subset: obsoletion_candidate
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:531775]: this "obsoletion_candidate" subset should be removed.

File: src/ontology/mondo-edit.obo [hunk 2]
```diff
@@ -531789,21 +531769,35 @@ is_a: MONDO:0016120 ! myotonic syndrome
...
+subset: otar {source="MONDO:OTAR"}
 subset: rare
+synonym: "DEL CASTILLO syndrome" EXACT [DOID:0050457]
 synonym: "Del Castillo syndrome" RELATED [OMIM:305700]
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:531779]: The same synonym (though different spelling) is both "exact" and "related". Please update to the correct synonym type (I don't know which one is correct, you'll have to research)

File: src/ontology/mondo-edit.obo [hunk 3]
```diff
@@ -531789,21 +531769,35 @@ is_a: MONDO:0016120 ! myotonic syndrome
...
-synonym: "Germinal cell aplasia" RELATED [OMIM:305700]
-synonym: "Sertoli cell-only syndrome" RELATED [OMIM:305700]
+synonym: "Germinal cell aplasia" EXACT [DOID:0050457, OMIM:305700]
+synonym: "Sertoli cell-only syndrome" EXACT [MONDO:0010595, OMIM:305700]
 synonym: "spermatogenic failure, X-linked, 1" RELATED [MONDO:Lexical, OMIM:305700]
 synonym: "spermatogenic failure, X-linked, type 1" EXACT [MONDORULE:1, OMIM:305700]
```
Comment (sabrinatoro) [inline src/ontology/mondo-edit.obo:531783]: same here, the same synonym, though spelled differently, is both related and exact. Please update as needed.

**Review by sabrinatoro — CHANGES_REQUESTED** [review]
@katiermullen, a few changes requested. Please see comments in line. Thank you.

File: src/ontology/mondo-edit.obo (hunk 2 above)
Comment (katiermullen) [reply src/ontology/mondo-edit.obo:531779]: @sabrinatoro I am getting myself a bit confused here and could kindly use your help: 

DO has the following: 
Screenshot 2024-10-29 at 11 31 27 AM

And lists Del Castillo Syndrome and Germinal Cell Aplasia as synonyms of the parent term, Sertoli cell-only syndrome. 

[OMIM:305700](https://omim.org/entry/305700) is Spermatogenic failure, x-linked, 1; SPGFX1 

Screenshot 2024-10-29 at 11 36 06 AM

Do you have advice on how to reconcile this?

File: src/ontology/mondo-edit.obo (hunk 1 above)
Comment (katiermullen) [reply src/ontology/mondo-edit.obo:531775]: done

File: src/ontology/mondo-edit.obo (hunk 3 above)
Comment (katiermullen) [reply src/ontology/mondo-edit.obo:531783]: updated to exact synonyms

File: src/ontology/mondo-edit.obo (hunk 2 above)
Comment (katiermullen) [reply src/ontology/mondo-edit.obo:531779]: updated to exact synonyms. updated db-xref for DOID:0050457 'Sertoli cell-only syndrome' to MONDO:mondoIsNarrowerThanSource

**Comment by katiermullen** [comment]
@sabrinatoro please review again and kindly let me know if changes are needed. Thank you for all of your help!

**Review by sabrinatoro — APPROVED** [review]
looks great, @katiermullen 
Approving and merging.

**Pull request merged**